src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.Unix.targets

```diff
@@ -74,8 +76,16 @@ The .NET Foundation licenses this file to you under the MIT license.
       <NativeFramework Include="GSS" />
     </ItemGroup>
 
+    <Message Text="IlcPath = $(IlcPath)" />
+    <Message Text="IlcHostPackagePath = $(IlcHostPackagePath)" />
+
+    <Exec Command="$(IlcPath)/source_code/libs/System.Globalization.Native/build-nativeaot-shim-standalone.sh $(IlcPath)/../ $(IntermediateOutputPath)" Condition="'$(LocalSystemGlobalizationNative)' != '' and '$(IlcPath)' != ''"/>
```

Comment:
You can assume that `IlcPath` / `IlcHostPackagePath` is always set. No need to check it for null.